index.js

```diff
@@ -78,7 +83,7 @@ class Plugin extends BtpPlugin {
   sendTransfer () {}
 
   async _createOutgoingChannel () {
-    debug('creating outgoing channel')
+    this._log.trace('creating outgoing channel')
```

Comment:
Creating an outgoing channel is an important step so this should probably be `info`